src/Docs/Session/Create/Check/RequestedLivenessCheckConfig.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yoti\Docs\Session\Create\Check;
+
+class RequestedLivenessCheckConfig implements RequestedCheckConfigInterface
```

Comment:
To match Java (perhaps not critical)
```suggestion
class RequestedLivenessConfig implements RequestedCheckConfigInterface
```